rosidl_typesupport_opendds_cpp/resource/srv__type_support.cpp.em

```diff
@@ -104,24 +113,126 @@ void * create_requester__@(service.namespaced_type.name)(
     DDS::Subscriber_var dds_subscriber,
     allocator_t allocator)
 {
-@# TODO: Implement, considering original code in ffe10f9 or earlier
+  @# TODO: remove next line when ready
```

Comment:
Is the plan to merge the PR with this line and the next line (and similar pattern in other functions) still here?

Reply:
Yes. Will uncomment before integration test, when RMW and Type Support is ready.